avstack/modules/perception/fov_estimator.py

```diff
@@ -97,6 +99,8 @@ def __call__(
             z_min=self.z_min,
             z_max=self.z_max,
         )
+        
+        self._eliminate_isolated_pts(pc_bev, 10, 30)
```

Comment:
Can you make `10` and `30` input parameters to the model so we could tune them if needed? Also provide a description of what they are in the docstring.